pom.xml

```diff
@@ -150,13 +150,18 @@
             <dependency>
                 <groupId>org.apache.tomcat.embed</groupId>
                 <artifactId>tomcat-embed-core</artifactId>
-                <version>11.0.5</version>
+                <version>11.0.6</version>
```

Comment:
Strongly recommend using a variable in parent pom.xml to avoid version conflicts in multiple child pom.xml.